src/fragments/lib/interactions/js/getting-started.mdx

```diff
@@ -76,7 +121,7 @@ Amplify.configure(awsconfig);
 
 ## ChatBot UI component
 
-Use the `ChatBot` component to add conversational UI to your app. [Learn more](/ui-legacy/interactions/chatbot).
+Use the `ChatBot` component to add conversational UI to your app. [Learn more](https://ui.docs.amplify.aws/react/legacy-components/chatbot).
 
 ## API reference
```

Comment:
you can remove the API reference section - in its current form - I feel it is more confusing than helpful. I am doing the same for other sections of our docs.

Reply:
ahh okay, makes sense
Removed API reference section